app/konnect/dev-portal/applications/dynamic-client-registration/okta.md

```diff
@@ -57,27 +57,34 @@ In order to map an application from the Dev Portal to Okta, you have to create a
 
 ## Configure the Dev Portal
 
-Once you have Okta configured, you can set up the Dev Portal to use Okta for dynamic client registration (DCR).
+Once you have Okta configured, you can set up the Dev Portal to use Okta for dynamic client registration (DCR). This will require two flows, one to create the DCR Provider, and one to create the Auth Strategy. DCR Providers are standalone configurations, meaning after you set up the Okta DCR Provider, you can reuse that config in many auth strategies.
```

Comment:
```suggestion
Once you have Okta configured, you can set up the Dev Portal to use Okta for dynamic client registration (DCR). This will require two flows, one to create the DCR provider, and one to create the auth strategy. DCR providers are standalone configurations, meaning after you set up the Okta DCR provider, you can reuse that config in other auth strategies.
```